src/main/java/fi/aalto/cs/apluscourses/presentation/ideactivities/TutorialViewModel.java

```diff
@@ -6,7 +6,9 @@
 import fi.aalto.cs.apluscourses.model.task.ActivityFactory;
 import fi.aalto.cs.apluscourses.model.task.Task;
 import fi.aalto.cs.apluscourses.utils.APlusLocalizationUtil;
+import java.awt.Frame;
 import java.util.List;
+import javax.swing.JOptionPane;
```

Comment:
I don't take pleasure in seeing AWT/Swing classes in view model classes... :)

Reply:
Me neither, but as a read-only field? Only used for comparison? Pretty please? :3

Jokes aside, this will be removed and replaced with WindowManager